packages/components/src/Table/TableWithDnD.test.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import TestBackend from 'react-dnd-test-backend';
+import { DragDropContext as dndContext } from 'react-dnd';
+import TestUtils from 'react-dom/test-utils';
+import Table from './Table';
+import TableCell from './Cell/TableCell';
+import draggable from './Cell/DraggableComponent';
+import * as TestData from './TestData';
+
+const elements = [TestData.element1, TestData.element2];
+
+function getComponentByName(components, name) {
+	return components.find(comp => comp.props.element.name === name);
+}
+
+/**
+ * Wraps a component into a DragDropContext that uses the TestBackend.
+ */
+function wrapInTestContext(DecoratedComponent) {
+	return dndContext(TestBackend)(props => <DecoratedComponent {...props} />);
+}
+
+const draggableCell = draggable(TableCell);
+
+/**
+ * This tests the drag and drop of the element1 onto the element2.
+ */
+it('drag-and-drop-on-table', () => {
```

Comment:
Put that is Talend.test.js please

Reply:
Sorry i don't understand. I have to rename the file? Or i have to move this test elsewhere?

Reply:
I suggest to move this test in `Table.test.js` (sorry typo in my first comment).

Reply:
ok